docs/application/native/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,258 @@
+     ```
+
+     When `component_id` is no longer needed, release it using `free()`.
+
+   - Get the application ID from the context:
+
+     ```
+     char *app_id;
+     ret = component_context_get_app_id(component_context, &app_id);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE)
+     	/* Error handling */
+     ```
+
+     When `app_id` is no longer needed, release it using `free()`.
+
+   - Verify whether the component with the given component context is terminated:
+
+     ```
+     bool terminated = false;
+
+     ret = component_context_is_terminated(component_context, &terminated);
+
+     if (false == terminated)
+         /* Application is running */
+     else
+         /* Application is terminated */
+     ```
+
+   - Clone the given context handle:
+
+     ```
+     component_context_h component_context_cloned = NULL;
+
+     ret = component_context_clone(&component_context_cloned, component_context);
+     ```
+
+   - Get the state of the component:
+
+     ```
+     component_state_e state;
+
+     ret = component_context_get_component_state(component_context, &state);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE)
+         /* Error handling */
+
+     if (state == COMPONENT_STATE_RESUMED)
+         /* Component is visible */
+     else
+         /* Component is invisible */
+     ```
+
+   - Verify whether the component is running as a sub component of the group:
+
+     ```
+     bool is_subcomponent = false;
+     ret = component_context_is_subcomponent(component_context, &is_subcomponent);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE)
+         /* Error handling */
+     ```
+   - Resume the running component:
+
+     ```
+     ret = component_manager_resume_component(component_context);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE)
+         /* Error handling */
+     ```
+     When the component is frame-component, it will appear on the screen.
+
+   - Terminate the running component in the background:
+
+     ```
+     ret = component_manager_terminate_bg_component(component_context);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE)
+        /* Error handling */
+     ```
+
+3. When you no longer need the component context, call `component_context_destroy()` to remove the handle and release all resources to prevent memory leaks:
+
+   ```
+     ret = component_context_destroy(component_context);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE)
+         /* Error handling */
+   ```
+
+<a name="filter"></a>
+## Getting Component Information
+
+To get the installed information and its details, and to operate on the information:
+
+1. Get the information of the currently-installed component with `component_manager_get_component_info()`. Its parameters are the ID of the component from which the information is being obtained, and the handle (`component_info_h`) to the component information which is filled with the received information. When a component is not installed, it is impossible to get its information.
```

Comment:
Done

Reply:
I too had the same doubt, however after researching, I found that it is okay to use currently-installed because it is a compound adjective

Reference:  https://english.stackexchange.com/questions/231516/is-currently-installed-a-proper-compound-adjective

Reply:
@surbhig12 I see. Thanks for the search and the reference link.
So, "currently-installed" is acceptable, the suggestion for here is :

 Get the information of the currently-installed component with `component_manager_get_component_info()`. Use the obtained component ID and `component_info_h` handle of the component with received information as the parameter. 
To get a component's information, the component must be installed.


If I'm wrong, please fix it.